README.md

```diff
@@ -8,11 +8,11 @@
 阿伦特《人的条件》说：
 “现代人缺失的正是劳动的公共性。”
 在公共性里，我才生产我自己，因而生产我要的社会。
-没得到这样劳动经验的人，是奴隶。#1
+没得到这样劳动经验的人，是奴隶。<sup>[1]</sup>
```

Comment:
Symbol "#" is always related to an issue.

Reply:
good call